extensions/test/e2e/framework/executor/executor.go

```diff
@@ -76,15 +76,14 @@ func (e *defaultExecutor) ExecCommandInContainerWithFullOutput(ctx context.Conte
 // execWithOptions executes a command in the specified container,
 // returning stdout, stderr and error. `options` allowed for
 // additional parameters to be passed.
-func execWithOptions(ctx context.Context, options execOptions) (stdout, stderr string, err error) {
+func execWithOptions(options execOptions) (stdout, stderr string, err error) {
 	const tty = false
 	req := options.client.Kubernetes().CoreV1().RESTClient().Post().
 		Resource("pods").
 		Name(options.podName).
 		Namespace(options.namespace).
 		SubResource("exec").
-		Param("container", options.containerName).
-		Context(ctx)
```

Comment:
Why is that removed?

Reply:
Because there is no `Context(ctx)` any more and btw it never had an affect because we only invoked the `.URL()` function of `req`.

Reply:
Basically, removing the context from the request was part of the K8s v0.18.x work in favor of the "executing" functions like `Post`, `GET`, etc. accepting a context.

Reply:
Hm ok, but the `Post()` function above doesn't accept a `context`, so I'm confused a bit. Anyways, thanks for the explanation.

Reply:
Sorry for the confusion. The function which executes the request is `Do()`

https://github.com/kubernetes/client-go/blob/57f65489c11a4d4a978c93ab369c38f53156888a/rest/request.go#L952

Reply:
OK, thanks 👍